### ISSUE TITLE
Gstreamer ugly plugins

### ISSUE THREAD

**Issue by TheDrifter363** [flathub/net.lutris.Lutris#279]
**Describe the problem**
The flatpak version of Lutris doesn't have the gstreamer plugins installed. I haven't verified if gstreamer bad, good, or base are also not installed. This causes games that require those decoders to work.

**Is it reproducible with native Lutris?**
No

**Steps to reproduce**
1. Install a game that requires the gstreamer ugly plugins like mpeg2, Silent Hill 3 is one example.
2. Play the game, the in game videos won't work.

**Expected behavior**
The in game videos to play.

**Actual behavior**
No in game videos play, though audio is present.

**Flatpak info**
`Flatpak 1.12.4`
`default host`

I believe this is the missing codec: libgstmpeg2dec.so for my specific game.

**Comment by strycore** [comment]
For Silent Hill 3, I add to install the 32bit version of libva-drm2, we should get this in the Flatpak.

**Comment by TheDrifter363** [comment]
Yeah I remember Arch having this same issue since they don't have the bad and ugly 32 bit gstreamer plugins available in the repos, you have to install them from the AUR. Hopefully someone on lutris will package these libraries. It'd make old games with these movies playable.

**Comment by strycore** [comment]
The GStreamer good, bad and ugly are metapackages and it doesn't really make sense to have them on distributions that have different licensing requirement than Debian. 
All the required codecs should be available in all major distros but their naming might differ. 
It's also important to note that we ship a lot of GStreamer libraries with our Wine builds but it seems we only ship 64bit libs for most of those.

**Comment by TheDrifter363** [comment]
So the main problem I'm having here is I use the Steam Deck, so I can't install any external libraries. It's why I was hoping the Lutris flatpak would have these gstreamer plugins built in. Wait, so it looks like you're a developer of Lutris? Is this something in the works?

I noticed the same issue with proton GE, they're not shipping the gstreamer ugly or bad plugins with their proton/wine versions. I should probably file a bug report there too.

Wait, I don't know much about how Arch handles licenses, but I thought Debian was pretty strict about non free software. It's why they have the 32 bit gstreamer bad and ugly plugins under non-free or contrib. I thought the reason Arch never put these in their main repository was because of manpower.

**Comment by CortexPE** [comment]
Bumping this issue, it breaks `GTA San Andreas` (32 bit) as well.
